### DISCUSSION TITLE
The smoothness is unreal!

### DISCUSSION THREAD

**Discussion by Espionage724** [diasurgical/DevilutionX#7580] [General]
I don't know how it's done, but the cursor input and just walking across Town on DevilutionX is the **smoothest** experience I've ever seen with a game. The cursor glides across the screen in low-latency perfection, and everything on the map moving with the screen happens with a level of sync I didn't think possible. And it's consistent Windows, Linux, even FreeBSD, out-the-box without any settings tweaking! :D

DevilutionX is awesome!

**Comment by julealgon** [comment]
Honestly DevilutionX has better frame handling/pacing than many AAA games these days 😆.

The only things it could still do in this regard would be to pre-scale assets to improve camera scrolling smoothness and to perform sub-pixel animation (although that can add a bit of blurring so that one is only subjectively better).

**Comment by Chonkblonk** [comment]
I am extremely pleased with DevX myself, having such a high quality port for one of my lifetime favorite games is a true luxury.
